app/models/concerns/template_tax.rb

```diff
@@ -0,0 +1,20 @@
+module TemplateTax
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def taxonomy_exportable
+      {
+        :organizations => method(:org_names),
+        :locations => method(:loc_names)
+      }
+    end
+
+    def org_names(template)
```

Comment:
since these are titles in fact, I think it should be called `org_titles` or even a bit more descriptive, `assigned_organizations_titles`, one other nice thing would be to accept user as parameter defaulting to User.current, so the method is reusable in other context